content/soarian/dstu2.md

```diff
@@ -53,6 +53,15 @@ For example:
 Where `https://fhir-myrecord.sandboxcerner.com/dstu2/abcd1234/` is the the `:serviceRootURL`, `Condition` is the 
 `:resource`, and `patientId=4342008` is the `:parameters`.
 
+### Secure Sandbox
+
+The secure sandbox instance can be used for testing an application with [authorization](#authorization). The service root URL for the sandbox instance is:
+
+`https://fhir-myrecord.sandboxcerner.com/dstu2/0e885770-571b-4c0c-b30f-21df9a058d0d`
+
+Example: `https://fhir-myrecord.sandboxcerner.com/dstu2/0e885770-571b-4c0c-b30f-21df9a058d0d/Condition?patientId= D4292B7B-AEDF-4CEF-B783-BD3AA1B0DD27`
```

Comment:
there's a space in this URL

Reply:
Removed space in the URL

Reply:
Changes are present here. https://github.com/cerner/fhir.cerner.com/pull/232/commits/416d911dabbb8e9fdd2602dbf306ae20ee43e10d